api/include/opentelemetry/context/runtime_context.h

```diff
@@ -125,5 +194,116 @@ inline Token::ContextDetacher::~ContextDetacher()
   token.context_ = context_;
   context::RuntimeContext::Detach(token);
 }
+
+// The ThreadLocalContextStorage class is a derived class from RuntimeContext and
+// provides a wrapper for propogating context through cpp thread locally.
+// This file must be included to use the RuntimeContext class if another
+// implementation has not been registered.
+class ThreadLocalContextStorage : public RuntimeContextStorage
```

Comment:
Is the parent class `RuntimeContext` instead of `RuntimeContextStorage`?

Reply:
I extracted parts that are needed to implement a custom `RuntimeContext` into a `RuntimeContextStorage` interface, which basically only handles the storage of context frames. I think this makes the whole design easier to understand.

Reply:
Fixed.

Reply:
nit: this comment can be removed now that we have single file: 
`This file must be included to use the RuntimeContext
// class if another implementation has not been registered.`

As this file needs to be always included.

Reply:
Changed already.